examples/testing-dom__clipboard/cypress/integration/permissions-spec.js

```diff
@@ -0,0 +1,62 @@
+/// <reference types="cypress" />
+
+// Permissions API specifically for clipboard information
+// https://web.dev/async-clipboard/
+// https://developer.mozilla.org/en-US/docs/Web/API/Permissions/query
+
+/* eslint-env browser */
+describe('Clipboard permissions', () => {
+  it('are granted in Electron', { browser: 'electron' }, () => {
+    cy.visit('index.html') // yields the window object
+    .its('navigator.permissions')
+    // permission names taken from
+    // https://w3c.github.io/permissions/#enumdef-permissionname
+    .invoke('query', { name: 'clipboard-read' })
+    .its('state')
+    .should('equal', 'granted')
+  })
+
+  it('can be queried in Chrome', { browser: 'chrome' }, () => {
+    cy.visit('index.html') // yields the window object
+    .its('navigator.permissions')
+    // permission names taken from
+    // https://w3c.github.io/permissions/#enumdef-permissionname
+    .invoke('query', { name: 'clipboard-read' })
+    .its('state').should('be.oneOf', ['prompt', 'granted', 'denied'])
+  })
+
+  it('can be granted in Chrome', { browser: 'chrome' }, () => {
```

Comment:
This example is confusing, it makes it seem like you can test clipboard in Chrome by overwriting the permissions, but you say it's impossible due to the popup. I would remove this. What's the use of this test practically?

Reply:
Yup, in Chrome we can grant the permissions using automation CDP call